h264/bits/bitreader.go

```diff
@@ -155,3 +157,14 @@ func (br *BitReader) PeekBits(n int) (uint64, error) {
 	r := (br.n >> uint(bits-n)) & ((1 << uint(n)) - 1)
 	return r, nil
 }
+
+// ByteAligned returns true if the reader position is at the start of a byte,
+// and false otherwise.
+func (br *BitReader) ByteAligned() bool {
+	return br.bits == 0
+}
+
+// BytesRead returns the number of bytes that have been read by the BitReader.
+func (br *BitReader) BytesRead() int {
```

Comment:
This doesn't seem necessary. You can either query the underlying reader or just do the accounting by the caller.

Reply:
So the caller should query the underlying reader for this information ? Is there an io.Reader implementation that provides this information ?

Reply:
I think this is stupid, but I'm just trying not to break the current code in this PR. If I get rid of this, and we do the querying by the caller, there's going to be alot more code added to this PR.

Reply:
OK. Leave it as is. Why do we need to know the number of bytes that were read?